op-dispute-mon/mon/bonds/bonds.go

```diff
@@ -0,0 +1,45 @@
+package bonds
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+
+	faultTypes "github.com/ethereum-optimism/optimism/op-challenger/game/fault/types"
+	"github.com/ethereum-optimism/optimism/op-service/sources/batching"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/holiman/uint256"
+	"golang.org/x/exp/maps"
+)
+
+var noBond = new(uint256.Int).Sub(uint256.NewInt(0), uint256.NewInt(1)).ToBig()
```

Comment:
The bond amounts in the [`FaultDisputeGame` contract](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol#L602-L603) are actually set to `type(uint128).max` once claimed.

I think we should just be able to use `new(big.Int).Sub(new(big.Int).Lsh(big.NewInt(1), 128), big.NewInt(1))`. This also gets rid of the need for the `github.com/holiman/uint256` library.

So
```suggestion
var noBond = new(big.Int).Sub(new(big.Int).Lsh(big.NewInt(1), 128), big.NewInt(1))
```

Reply:
I've created a pr into this branch implementing this change to demonstrate the complete diff :) https://github.com/ethereum-optimism/optimism/pull/9463. Should be good to merge into here and then merge into trunk

Reply:
Ugh, 128 vs 256, I do that all the time... Thanks.